cli/src/lib/compiler/compile.js

```diff
@@ -94,8 +94,19 @@ const compile = async ({
     }
     const compileFile = async (source, destination) => {
         if (source.match(extensionPattern)) {
-            const result = await babel.transformFileAsync(source, babelConfig)
-            await fs.writeFile(destination, result.code)
+            try {
+                const result = await babel.transformFileAsync(
+                    source,
+                    babelConfig
+                )
+                await fs.writeFile(destination, result.code)
+                console.info(`Compiling ${source}: OK.`)
+            } catch (err) {
+                console.error(err)
+                console.info(
+                    `Compiling ${source}: ERROR. Fix the problem and save the file to automatically reload.`
+                )
```

Comment:
The console messages here are not final, but they can be helpful for the user so that they know _when_ the problematic file is OK again.

Reply:
Should these use the `reporter` logger for consistency?

One suggestion: using the the file's relative path from the working directory would keep the logs nicer; `source` seems to be an absolute path which can be quite long.

In a project with lots of files, these logs can be quite verbose at startup of the dev server when all the file paths in the project logged -- not functionally critical, but it would be nice aesthetically if those can be skipped

Reply:
> In a project with lots of files, these logs can be quite verbose at startup of the dev server when all the file paths in the project logged -- not functionally critical, but it would be nice aesthetically if those can be skipped

Agreed. If we remove the "OK" message now, then the user won't get a message that the compilation succeeded, and the last message in the terminal will be an error message, even though the file has recompiled OK and is ready to go. Probably not worth the upfront spam though.

Reply:
> Should these use the `reporter` logger for consistency?

Yes, definitely. I left them as console logs to make it clear that these are not final, for the above reasons. :)

Reply:
Removed the "OK" message because of the spam.

Changed to a relative path print for `source`.

Changed to use reporter.